dask_ml/datasets.py

```diff
@@ -64,12 +64,14 @@ def make_counts(
     rng = dask_ml.utils.check_random_state(random_state)
 
     X = rng.normal(0, 1, size=(n_samples, n_features), chunks=(chunks, n_features))
-    informative_idx = rng.choice(n_features, n_informative, chunks=n_informative)
-    beta = (rng.random(n_features, chunks=n_features) - 1) * scale
+    informative_idx = rng.choice(
+        n_features, n_informative, chunks=n_informative, replace=False
+    )
+    beta = (rng.random(n_features, chunks=n_features) - 0.5) * 2 * scale
```

Comment:
Why is this change required?

Reply:
I had changed this due to some issues with convergence for the Poisson Regressor. I am not an expert in Poisson regression, but this previously had `beta` values between `[-1, 0)` and the update brought that to `[-1, 1]`. Reverting it back makes no difference in the updated tests, so I will revert the change.